python/pyspark/pandas/namespace.py

```diff
@@ -2747,13 +2747,19 @@ def merge(
 
 
 @no_type_check
-def to_numeric(arg):
+def to_numeric(arg, errors="raise"):
     """
     Convert argument to a numeric type.
 
     Parameters
     ----------
     arg : scalar, list, tuple, 1-d array, or Series
+        Argument to be converted.
+    errors : {'raise', 'coerce'}, default 'raise'
+        * If 'coerce', then invalid parsing will be set as NaN.
+        * If 'raise', then invalid parsing will raise an exception.
```

Comment:
sorry last comment. Seems like it's going to work for other types but not for pandas-on-Spark series. Can we add `ignrore` here with the note that it doesn't work for pandas-on-Spark series?